test/unit/math/rev/meta/vector_seq_view_test.cpp

```diff
@@ -0,0 +1,29 @@
+#include <stan/math/rev.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+
+template <typename T>
+void test_vec_seq_var(const T& m1) {
```

Comment:
Missing tests for `size` and `operator[]`

Reply:
I added a couple size checks [here](https://github.com/stan-dev/math/pull/2292/commits/d7124dff41dfeb7bc5fb58816d4658b21c8fdd78).